streams/src/main/java/org/apache/kafka/streams/state/internals/StoreChangeLogger.java

```diff
@@ -27,6 +27,14 @@
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * Store change log collector that batches updates before sending to Kafka.
+ *
+ * Note that array typed key may result in incorrect behavior as it has undefined hashing function.
```

Comment:
I'd add: "If you are working on byte arrays, for example, you may want to wrap them with the `Bytes` class."